docs/store-operations/translations/locations.mdx

```diff
@@ -0,0 +1,228 @@
+                                },
+                                {
+                                    "fieldName": "label",
+                                    "original": "LOCATION 1",
+                                    "translation": null
+                                },
+                                {
+                                    "fieldName": "address2",
+                                    "original": "apt 111",
+                                    "translation": null
+                                }
+                            ]
+                        },
+                        "cursor": "eyJpZCI6IjEifQ=="
+                    }
+                ]
+            }
+        }
+    }
+}
+```
+</Tab>
+</Tabs>
+
+### Update a translation
+
+This mutation updates a translation.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```json filename="Example mutation: Update a translation" showLineNumbers copy
+        GRAPHQL {{host}}/stores/{{store_hash}}/graphql
+        X-Auth-Token: {{token}}
+
+      mutation {
+        translation {
+            updateTranslations(input: {
+                resourceType: INVENTORY_LOCATIONS,
+                channelId: "bc/store/channel/1",
+                localeId: "bc/store/locale/es",
+                entities: [
+                {
+                    resourceId: "bc/store/inventoryLocation/1",
+                    fields: [
+                        {
+                            fieldName: "city",
+                            value: "Ville (OVR) TEST ES"
+                        },
+                        {
+                            fieldName: "state",
+                            value: "État (OVR) TEST ES"
+                        }
+                    ]
+                },
+                {
+                    resourceId: "bc/store/inventoryLocation/2",
+                    fields: [
+                        {
+                            fieldName: "city",
+                            value: "Ville (OVR) TEST ES"
+                        },
+                        {
+                            fieldName: "state",
+                            value: "État (OVR) TEST ES"
+                        }
+                    ]
+                }
+            ]
+        }) {
+            __typename
+            errors {
+                __typename
+                ... on Error {
+                    message
+                }
+            }
+        }
+    }
+}
+```
+  </Tab>
+<Tab>
+```json filename="Example mutation: Update a translation" showLineNumbers copy
+ {
+    "data": {
+      "translation": {
+        "updateTranslations": {
+          "__typename": "UpdateTranslationsResult",
+          "errors": []
+        }
+      }
+    }
+  }
+```
+</Tab>
+</Tabs>
+
+### Delete a translation
+
+The following mutation deletes a translation.
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+    ```json filename="Example mutation: Delete a translation" showLineNumbers copy
```

Comment:
what is mean `showLineNumbers copy`?

Reply:
Hi @bc-yehor-kravchenko This means to show the line numbers for the code and allow for copying all code at once.